Source/ablastr/fields/PoissonSolver.H

```diff
@@ -161,7 +161,7 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
     // scale rho appropriately; also determine if rho is zero everywhere
     amrex::Real max_norm_b = 0.0;
     for (int lev=0; lev<=finest_level; lev++) {
-        rho[lev]->mult(-1._rt/PhysConst::ep0); // TODO: when do we "un-multiply" this? We need to document this side-effect!
+        rho[lev]->mult(-1._rt/PhysConst::ep0);
```

Comment:
Proposal: let's remove this here, make `rho` a const member and explicitly scale up before we call `computePhi`?

Reply:
The advantage of doing the scaling here is that it reduces code duplication for most use cases.